src/Microsoft.AspNetCore.Razor.Runtime/project.json

```diff
@@ -1,5 +1,10 @@
 {
-  "description": "Runtime components for rendering Razor pages.",
+  "description": "Runtime components for rendering Razor pages and implementing tag helpers.\r\nCommonly used types:\r\nMicrosoft.AspNetCore.Razor.TagHelpers.HtmlAttributeNameAttribute\r\nMicrosoft.AspNetCore.Razor.TagHelpers.HtmlTargetElementAttribute\r\nMicrosoft.AspNetCore.Razor.TagHelpers.ITagHelper\r\nMicrosoft.AspNetCore.Razor.TagHelpers.TagHelper",
```

Comment:
Windows line endings don't seem to generally cause much issue on non-Windows. Plus, the main place it's ever seen is in NuGet UI, which is at least at the moment all Windows-only (or web based).